financialaid/management/commands/generate_exchange_rates_test.py

```diff
@@ -0,0 +1,47 @@
+"""
+Test for management command generating exchange rates
+"""
+from mock import patch
+
+from financialaid.constants import CURRENCY_EXCHANGE_RATE_API_REQUEST_URL
+from financialaid.management.commands import generate_exchange_rates
+from financialaid.models import CurrencyExchangeRate
+from search.base import ESTestCase
+
+
+@patch('financialaid.tasks.requests.get')
+class GenerateExchangeRatesTest(ESTestCase):
+    """
+    Tests for generate_exchange_rates management command
+    """
+    @classmethod
+    def setUpTestData(cls):
+        cls.command = generate_exchange_rates.Command()
+
+    def setUp(self):
+        super(GenerateExchangeRatesTest, self).setUp()
+        self.data = {
+            "extraneous information": "blah blah blah",
+            "rates": {
+                "CBA": "3.5",
+                "FED": "1.9",
+                "RQP": "0.5"
+            }
+        }
+
+    def test_currency_exchange_rate_command(self, mocked_request):
```

Comment:
Please add tests that simulate API failures. At a minimum, we should be able to handle a "500 Server Error" failure and a "429 Rate Limit Exceeded" failure. Also check out [the list of documented error responses](https://docs.openexchangerates.org/docs/errors).